src/site/content/en/fast/incorporate-performance-budgets-into-your-build-tools/index.md

```diff
@@ -40,10 +69,10 @@ The default limit for both assets and entry-points is **250 KB**. You can set yo
   </figcaption>
 </figure>
 
-The numbers are compared against **uncompressed asset sizes**. This is not an ideal situation, since most hosting platforms, CDNs and reverse proxy servers compress assets by default. You can give yourself some wiggle room during development, but keep in mind that compression speeds up only the transfer. Browsers still have to parse uncompressed files and this [parsing cost is not small](https://medium.com/@addyosmani/the-cost-of-javascript-in-2018-7d8950fbb5d4), especially on mobile devices.
+The numbers are compared against **uncompressed asset sizes**. This is not an ideal situation, since most hosting platforms, CDNs and reverse proxy servers compress assets by default. You can give yourself some wiggle room during development, but keep in mind that compression speeds up only the transfer. Browsers still have to execute uncompressed files and the [cost of this is not small](https://v8.dev/blog/cost-of-javascript-2019), especially on mobile devices.
```

Comment:
```suggestion
Budgets are compared against **uncompressed asset sizes**. This is not an ideal situation, since most hosting platforms, CDNs and reverse proxy servers compress assets by default. You can give yourself some wiggle room during development, but keep in mind that compression speeds up only the transfer. Browsers still have to execute uncompressed files and the [cost of this is not small](https://v8.dev/blog/cost-of-javascript-2019), especially on mobile devices.
```

Reply:
I think I'd rework this paragraph a bit. I don't think comparing again uncompressed asset sizes is necessarily bad - it's just that compressed asset sizes typically correlate with transfer times; while uncompressed asset sizes correlate with parse/compile/execution times.

Reply:
We're not really saying it's bad, it's "not ideal" and we acknowledge that uncompressed sizes affect execution time. How about:

> "You can give yourself some wiggle room during development, but that doesn't mean setting lax budgets and counting on compression to kick in. Keep in mind that compression speeds up only the transfer and browsers still have to execute uncompressed files..."

Reply:
Ah, so I’d probably argue that using uncompressed asset sizes really isn’t that bad and is actually totally fine.

There’s a ton of variation, so it really depends on a particular person and their device + network, but generally parse/compile/execute is now a bigger issue than network times, which would correlate more with uncompressed sizes. However, historically, networks have been the biggest issue, which I think is why there’s still a tendency for budgets and tooling to be based around transfer size rather than “actual” size. Neither metric (compressed/uncompressed) is perfect, but both have their uses.

Reply:
This was discussed when I was first writing this guide. At the time, the consensus was that, because our own tooling and budgeting guidelines are based on compressed sizes, we want to echo that preference here. Pretty sure there was some talk about nudging webpack to adopt that approach too.

I'm fine with us being less opinionated, but let's consult @addyosmani ?

Reply:
SGTM. @addyosmani thoughts on this?

Reply:
I'm also okay with us being less opinionated in the short term. Until we are at a place where there's more consistency across perf budget tooling in terms of whether they're looking at compressed vs uncompressed numbers, this seems like a fair compromise.